docs/internationalization/core.md

```diff
@@ -153,6 +153,8 @@ The translation strings for repairs issues are defined under the `issues` key. A
 Integrations can provide translations for names of its entities. To do this, provide an `entity` object, that contains translations of the names and set the entity's `translation_key` property to a key under a domain in the `entity` object.
 If the entity's `translation_key` property is not `None` and the `entity` object provides a translated name, `EntityDescription.name` will be ignored.
 
+Entity components provide shared translations which should be used when suitable to avoid translating the same thing multiple times.
```

Comment:
I'm not so happy about the wording here, how can we explain better what's provided by entity component integrations and when to use those translations?

Reply:
I think the gist is good, but maybe the context is not working toward someone's imagination. Suggestion:

```suggestion
Entity components, like `sensor`, already have existing translations available that can be reused by referencing those. This includes common translations for entity names based on a device class. For example, it already has translations available for a "Temperature" sensor that can be referenced. Referencing existing translations is preferred, as it prevents translating the same thing multiple times.
```